src/main/kotlin/com/meltwater/docker/compose/DockerCompose.kt

```diff
@@ -200,16 +257,22 @@ class DockerCompose private constructor(
 
     private fun verifyDockerComposeInstallation() {
         try {
-            val versionOutput = executeCommand("docker-compose --version")
+            val versionOutput = if (dockerCompose == "docker compose") {
+                executeCommand("$dockerCompose version")
+            } else {
+                executeCommand("$dockerCompose --version")
+            }
```

Comment:
There is a slight difference in invocation between "docker-compose" and "docker compose". The former implements `version` as a flag, while the latter implements it as a subcommand.